src/HttpReply.cc

```diff
@@ -612,49 +612,45 @@ void HttpReply::removeStaleWarnings()
     }
 }
 
-/**
- * Remove warning-values with warn-date different from Date value from
- * a single header entry. Returns a string with all valid warning-values.
- */
-String HttpReply::removeStaleWarningValues(const String &value)
+/// \returns a list of warning-values with warn-date equal to the reply Date
+String
+HttpReply::findFreshWarningValues(const String &warningFieldValue) const
 {
     String newValue;
     const char *item = 0;
     int len = 0;
     const char *pos = 0;
```

Comment:
Before merge, please correct these to nullptr initialization. This is technically a new function even though it shares a lot of syntax with the old one.